.github/workflows/linux.yml

```diff
@@ -265,6 +265,11 @@ jobs:
             configure: --enable-debug --disable-shared --disable-threaded-resolver --with-libssh --with-openssl
             tflags: -n --test-duphandle '!TLS-SRP'
 
+          - name: libssh valgrind
+            install_packages: libssh-dev valgrind
+            configure: --enable-debug --with-libssh --with-openssl
+            tflags: scp sftp
+
```

Comment:
Would it have a downside to integrate this in the existing `openssl -O3 valgrind` job?
It could save some time by avoiding the extra build.

Reply:
If we can switch that one to libssh, then no!